WordPress/Sniffs/Theme/NoFaviconSniff.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+/**
+ * Check for hardcoded favicons instead of using core implementation.
+ *
+ * @link    https://make.wordpress.org/themes/handbook/review/required/#core-functionality-and-features
+ *
+ * @package WPCS\WordPressCodingStandards
+ *
+ * @since   0.xx.0
+ */
+class WordPress_Sniffs_Theme_NoFaviconSniff implements PHP_CodeSniffer_Sniff {
+
+	const REGEX_TEMPLATE = '` (%s)`i';
+	const REGEX_ATTR_TEMPLATE = '%1$s=[\'"]%2$s[\'"]';
```

Comment:
Similar to how you document the class properties. I'm not sure about the tag to use at the moment - I believe at some point there was a `@const` tag, but would need to check if that's still a thing.
To check I'd have a look through the current version of the PSR proposal for documentation and at the PHPDocumentor documentation.